config/saml2.php

```diff
@@ -137,6 +137,16 @@
 
     'proxyVars' => false,
 
+    /*
+    | (Optional) Which class implements the route functions.
+    | If left blank, defaults to this lib's controller (Slides\Saml2\Http\Controllers\Saml2Controller).
+    | If you need to extend Saml2Controller (e.g. to override the `login()` function to pass
+    | a `$returnTo` argument), this value allows you to pass your own controller, and have
+    | it used in the routes definition.
+    |
+    */
+    'saml2_controller' => '',
```

Comment:
Why not define the default one here?

Reply:
I went with the empty string so as to mirror the original PR as closely as possible
https://github.com/aacotroneo/laravel-saml2/commit/34fe227c2683dd8ed2a2b439731bb5a1755841b7

I imagine the original author probably chose to do this so that user error would be marginally safer (that is to say, if they set their config to false or something it shouldn't break the routes).